ci-operator/step-registry/multiarch/tuning-operator/deploy/multiarch-tuning-operator-deploy-commands.sh

```diff
@@ -15,6 +15,17 @@ function debug() {
         echo -e "oc -n $NAMESPACE get deployments -owide\n$(oc -n $NAMESPACE get deployments -owide)"
         echo -e "Getting pod info....\n"
         echo -e "oc -n $NAMESPACE get pods -owide\n$(oc -n $NAMESPACE get pods -owide)"
+        echo -e "Getting namespace events...\n"
+        echo -e "oc -n $NAMESPACE get events\n$(oc -n $NAMESPACE get events)"
+        echo -e "Getting unhealthy pod logs...\n"
+        oc get pod -n $NAMESPACE --no-headers | awk '{print $1" "$3}' | while read line; do
+          name=$(echo $line | awk '{print $1}')
+          status=$(echo $line | awk '{print $2}')
+          if [[ $status != "Completed" && $status != "Running" ]]; then
+            echo "Getting pod log of $name, status is $status"
+            oc logs -n $NAMESPACE $name
+          fi
+        done
```

Comment:
Hi @qiliRedHat, With the recent changes, we are now exposing some labels that might help understand the operator's behavior at scale.

See https://github.com/openshift/multiarch-tuning-operator/pull/247

If I understand correctly, node density is meant to create a constant number of pods per second in the cluster under testing.

As part of the perf/scale efforts, we would like to know:


1. How many pods did get the scheduling gate set and removed? `multiarch.openshift.io/scheduling-gate: removed`
2. How many pods did NOT get the scheduling gate set? `multirach.openshift.io/scheduling-gate` shouldn't be set. This value should be 0.
3. How many pods did get the node affinity set? `multirach.openshift.io/node-affinity: set`. This value should be equal to the one in (1)
4. How many pods did NOT get the the node affinity set? This should be equal to (2)=0=(3)-(1)
5. Any left pending pod? `multiarch.openshift.io/scheduling-gate: gated` or the status.Phase == pending with reason "SchedulingGated".


If the pods are not deleted at the end of the testing, we might get these information here too.

Reply:
cc @lwan-wanglin

Reply:
yes, we can add junits for that

Reply:
Yes, I think it is possible to configure to not let the pods being deleted at the end of the testing. I'll check how to do it and update the jobs.

Reply:
@aleskandro I have added a env `GC: "false"` to not deleting the test workload after the `openshift-qe-node-density` test.
@lwan-wanglin If you have added the junits to check the measures, please let me know what test I can add to the end of the perfscale test `openshift-qe-node-density`.

Reply:
@lwan-wanglin The label of the test pods is `kube-burner-job=node-density`. If possible we can make it a env var to be more portable to other tests.

Reply:
talked with @qiliRedHat in private, I will take the following labels calculation and junits creation.